src/fuzzer.py

```diff
@@ -184,19 +205,22 @@ def fuzzing_round(self, test_case: TestCase, inputs: List[Input]) -> Optional[Eq
 
         # Violation survived priming. Report it
         return violation
-
-    def boost_inputs(self, inputs: List[Input], nesting: int) -> List[Input]:
-        if CONF.inputs_per_class == 1:
-            return inputs
-
+    
+    def boost_inputs(self, inputs: List[Input], nesting: int) -> tuple[List[CTrace], List[Input]]:
```

Comment:
It should be `Tuple` (datatype) instead of `tuple` (class). I'm not completely sure what is the difference between them, to be honest, but that's what people normally use.

Reply:
Also, perhaps now that `boost_inputs` returns contract traces, would it make sense to rename it to something like `trace_and_boost`?

Reply:
Done as suggested.